frontend/website/pages/docs/developers/sandbox-container.mdx

```diff
@@ -0,0 +1,47 @@
+import Page from '@reason/pages/Docs';
+export default Page({title: "Sandbox Container"});
+
+# Sandbox Container
+
+If all you need is a running daemon and some blocks, the sandbox container has everything you need! It uses the same configuration as the testnet, but instead of the community participants ledger it uses a simple ledger with a single demo account.
+
+This can be useful for testing or playing with some of the core features of the protocol.  This node supports multiple accounts, sending transactions between them, and will also support delegating and staking. In fact since it's a single node network, you'll get all the block rewards!
+
+<Alert>
+
+This container will **NOT** connect you to the live network.
+
+</Alert>
+
+## Installation
+
+In order to use the demo container, you'll need to install Docker, the simplest way will be to use [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+After you have Docker installed run the following command to spin up the container.
+
+```
+docker run --publish 3085:3085 -it \
```

Comment:
I'm not sure what level of developer you are targeting but I think it's at least worth explaining running in interactive mode in the foreground as I can foresee confusion things stopping when the terminal is closed. 

To a sample command, assuming lots will copy/paste I'd at least add `--name coda` (or whatever) so you can demonstrate following commands without docker knowledge.

Perhaps an additional mention of running detached (-d) for a long-running process in addition to tailing logs `docker logs --follow coda`?